spec/functions/lookup_spec.rb

```diff
@@ -58,6 +58,13 @@
     }.to raise_error(Puppet::Error, %r{Unable to parse a hostname})
   end
 
+  it 'returns nil instead of raising when raising is disabled' do
+    expect {
+      result = function.execute('/v1/whatever', 'vault.docker', false)
+      expect(result).to be(nil)
```

Comment:
Is it bad form to do additional things inside the `expect` block like this? If so, does the assertion for no error seem necessary or should I just check the return value (since the test will fail/error if the function errors anyway)?